### ISSUE TITLE
Use Curl BinaryBuilder artifact

### ISSUE THREAD

**Issue by activexray** [helgee/RemoteFiles.jl#24]
Instead of depending on the user having curl installed, you can have this package depend on https://github.com/JuliaWeb/LibCURL.jl and use it's executable instead

**Comment by helgee** [comment]
Yes, or I could just use the Downloads.jl stdlib which uses libCURL internally.
